src/MF_Segment/LedControl_dual.cpp

```diff
@@ -0,0 +1,492 @@
+// =======================================================================
+// @file        LedControl_dual.cpp
+//
+// @project     MobiFlight custom Firmware
+//
+// @author      GiorgioCC (g.crocic@gmail.com) - 2023-06-29
+// @modifiedby  GiorgioCC - 2023-07-09 20:13
+//
+// A library for controlling LED 7-segment displays with either
+// a MAX7219/MAX7221 or a TM1637 (4/6 digit) driver
+// Portions of code derived from:
+// - LedControl - A library for controlling Leds with a MAX7219/MAX7221
+//   Copyright (c) 2007 Eberhard Fahle
+// - TM1637TinyDisplay - TM1637 Tiny Display library by Jason A. Cox
+//   (https://github.com/jasonacox)
+//
+// =======================================================================
+
+#include "LedControl_dual.h"
+
+// =======================================================================
+// Common Definitions
+// =======================================================================
+
+// Segments to be switched on for characters and digits on 7-Segment Displays
+// bit/segment sequence: dABCDEFG
+//      A
+//     ---
+//  F |   | B
+//     -G-
+//  E |   | C
+//     ---   .d
+//      D
+
+const static byte charTable[128] PROGMEM = {
```

Comment:
We could also save some space if we don't use the first 4 lines. We would just to make sure that we are always requesting actual chracters > 32